react-app/src/app/components/Game/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+
+import Board from '../Board';
+
+import style from './styles.scss';
+
+class Game extends Component {
+  render() {
+    return (
+      <div className={style.game}>
+        <div>
+          <Board />
+        </div>
+        <div className={style.gameInfo}>
+          <div>{/* status */}</div>
+          <ol>{/* TODO */}</ol>
+        </div>
+      </div>
+    );
+  }
+}
+
+export default Game;
```

Comment:
Game should be an screen. `Then Board...Squares and all componentes should be inside Game screen.`

Reply:
For example: `/src/app/screens/Game/components/Square/index.js`